.github/workflows/cd.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   test-lint-publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
There have been issues with the github actions pushing to npm and the github pages. I've updated some of the permissions and hopefully upgrading these is enough to make it work again.

Reply:
I suppose we don't really have a way of testing that this will fix the issue, yeah? Although I guess if it's already broken there's not any harm 😅

Reply:
Yeah it really needs a commit to master to test it. I tried updating some permissions settings before merging in dependabot changes but it didn't fix it so I'm hopeful that the version update will do something 😬

Reply:
Also what I had been doing previously was to run the `npm publish` manually, but that doesn't update the github pages. At least for the package itself, I updated the version number here so that part will at least be fine.